js/src/common/states/AlertManagerState.ts

```diff
@@ -55,7 +55,7 @@ export default class AlertManagerState {
   /**
    * Dismiss an alert.
    */
-  dismiss(key: AlertIdentifier): void {
+  dismiss(key: AlertIdentifier | null): void {
```

Comment:
What's the motivation for this change? I see that we check for a falsy key below (maybe that should also be changed), but is there a use case for `null` to be provided as a key to dismiss?

Reply:
Not necessarily a use case, but if we check for null/falsy then we should probably show that we are handling said type... then exts don't need to do the check themselves in the code (even though it's barely any extra code). 🤷‍♂️

Reply:
Fair enough